app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from os import makedirs, remove
+
+
+def move_file(command: str):
+    command_parts = command.split()
+    source_file = command_parts[1]
+    second_file_path = command_parts[2]
+    destination_folder = second_file_path.rsplit("/", 1)[0]
+    makedirs(destination_folder)
```

Comment:
This function will produce an error if destination directory already exists. It can be changed with `exists_ok` parameter.